front/src/monthview/Monthview.js

```diff
@@ -1,7 +1,6 @@
 import React, { Component } from 'react';
 
 import '../App.css';
```

Comment:
Should not need to be imported as all classNames are declared with `classes(css.foo)` and shared.scss is imported in scss.
```suggestion
```